proposals/wg-auth.md

```diff
@@ -0,0 +1,70 @@
+# OCI Working Group Proposal: Authentication and Authorization
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+## Auth OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Auth Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”).
+The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter.
+Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+Authentication and authorization are a key requirement for registries to control access.
+Implementations of this in registries and clients have roughly followed standards set by Docker and have mostly focused on compatibility with specific implementations of registries or clients.
+This working group will define a standard to be supported by OCI compatible registries and clients when support for authentication and authorization is required.
+
+## Scope
+
+* Define registry responses to unauthenticated requests.
```

Comment:
Is this referring to the "anonymous" pull case where an authentication server is giving out tokens for unauthenticated requests?

Reply:
I'm thinking of requests where either there is no Authorization header, or the token in the header does not apply to the request (a pull token used to push, or a token for a different repository being reused for a future request).